### PR TITLE
style: Clean up FormText code and tests

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: test/FormTextSpec.js [hunk 1]
```diff
@@ -23,13 +23,8 @@ describe('<FormText>', () => {
...
-        .find('small')
-        .hasClass('form-text'),
-    ).to.equal(true);
-    expect(
-      mount(<FormText muted />)
-        .find('small')
+        .find('small.form-text')
         .hasClass('text-muted'),
-    ).to.equal(true);
+    ).to.be.true;
```
Comment (jquense) [inline test/FormTextSpec.js]: unused expressions?

Comment (taion) [reply test/FormTextSpec.js]: apparently we don't lint for it? we have `.should.be.true` elsewhere. should we use dirty-chai?

Comment (taion) [reply test/FormTextSpec.js]: apparently we don't lint for it? we have `.should.be.true` elsewhere. should we use dirty-chai?

Comment (jquense) [reply test/FormTextSpec.js]: probably, I don't think anyone felt like updating

**Pull request merged**